Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5

```
@@ -1311,6 +1311,9 @@
         "true_name.title": "Transgress Other",
         true_name: "I attempted to $(l:patterns/readwrite#hexcasting:write)$(action)save a reference/$ to another player to a permanent medium.$(br2)Causes black sparks, and robs me of my sight for approximately one minute.",
 
+        "stack_size.title": "Stack Size",
+        "stack_size": "I tried to cast a spell which exceeded the size limit of the $(thing)Stack/$.$(br2)Causes my spell to halt and pushes a $(l:casting/influences)$(action)Garbage/$ to my stack.",
```

```suggestion
        "stack_size.title": "Too Many Iotas",
        "stack_size": "I tried to cast a spell which exceeded the size limit of the stack.$(br2)Causes black sparks, and my entire stack will be replaced with a single $(l:casting/influences)$(action)Garbage/$.",
```

This should also be moved to just below the "Not Enough Iotas" mishap. i think i'm so funny